code/modules/food_and_drinks/plate.dm

```diff
@@ -75,6 +75,7 @@
 #define PLATE_SHARD_PIECES 5
 
 /obj/item/plate/throw_impact(atom/hit_atom, datum/thrownthing/throwingdatum)
+	. = ..()
```

Comment:
i wonder if `throw_impact()` should have `SHOULD_CALL_PARENT(TRUE)` (probably a separate project)